google/cloud/spanner_v1/transaction.py

```diff
@@ -185,7 +186,14 @@ def _make_params_pb(params, param_types):
         return {}
 
     def execute_update(
-        self, dml, params=None, param_types=None, query_mode=None, query_options=None
+        self,
+        dml,
+        params=None,
+        param_types=None,
+        query_mode=None,
+        query_options=None,
+        retry=gapic_v1.method.DEFAULT,
```

Comment:
I think having `retry` and `timeout` as [keyword-only arguments](https://www.python.org/dev/peps/pep-3102/) is tidier. If more arguments relating to the contents of the request are added later, it will be possible to group them with the other content arguments without a breaking change, e.g.:
```
    def execute_update(
        self,
        dml,
        params=None,
        param_types=None,
        query_mode=None,
        query_options=None,
        additional_feature=None
        *,
        retry=gapic_v1.method.DEFAULT,
        timeout=None,
    ):
```
WDYT?